tests/unit/core/testErrorHandler.js

```diff
@@ -263,5 +263,15 @@ describe('errorHandler', () => {
       const handler = new ErrorHandler();
       expect(handler.hasError()).toBe(false);
     });
+
+    it('returns no component if it does not have an error', () => {
+      const handler = new ErrorHandler();
+      expect(handler.renderErrorIfPresent()).toBe(null);
+    });
+
+    it('returns a component if it has an error', () => {
+      const handler = new ErrorHandler({ capturedError: new Error() });
+      expect(handler.renderErrorIfPresent()).not.toBe(null);
```

Comment:
Ah we [naughtily inject the enzyme testing wrapper into all our test files](https://github.com/mozilla/addons-frontend/blob/master/tests/setup.js#L6), so you should be able to just use that out of the box 😄